src/fiatExchanges/ExchangeQR.tsx

```diff
@@ -88,7 +88,7 @@ export default function ExchangeQR({ route, navigation }: Props) {
   return (
     <View style={styles.container}>
       <View style={styles.qrContainer}>
-        <StyledQRCode qrSvgRef={qrSvgRef} />
+        <StyledQRCode dataType={QRCodeDataType.Address} qrSvgRef={qrSvgRef} />
```

Comment:
any easy way to unit test this?

Reply:
I couldn't find an easy way to test this. We could add snapshots, but IDK if its worth that. I did add new unit tests to the hook so we have at least some additional security.